### DISCUSSION TITLE
Backticks are not removed when using Code extension.

### DISCUSSION THREAD

**Discussion by andersr** [ueberdosis/tiptap#4233] [General Questions]
I installed the [Code extension](https://tiptap.dev/api/marks/code).  However, in contrast to what appears in the demo, the backticks before and after the code block are not removed/hidden.

In other words, I am seeing this:

![Screenshot 2023-07-19 at 11 15 51 AM](https://github.com/ueberdosis/tiptap/assets/819213/0b5738af-d379-4040-81a7-aacbbf22a942)

Why are they appearing for me but not in the demo? How can I make them go away?

Thanks!

**Comment by ruthkangyr** [comment]
hi sorry, can I ask if you managed to remove the backticks and how you did it?

**Comment by yanhao-li** [comment]
they can be hidden with `[&::before]:content-none [&::after]:content-none`

For example:
```ts
const editor = useEditor({
    extensions: [
      Code.configure({
        HTMLAttributes: {
          class:
            'rounded-md bg-gray-100 px-1.5 py-1 font-mono text-sm [&::before]:content-none [&::after]:content-none',
        },
      }),
    ],
```
